src/amo/containers/App.js

```diff
@@ -20,9 +34,22 @@ export class AppBase extends React.Component {
           defaultTitle={i18n.gettext('Add-ons for Firefox')}
         />
         {children}
+        <footer>{this.logIn()}</footer>
       </div>
     );
   }
 }
 
-export default translate({ withRef: true })(AppBase);
+export const makeMapStateToProps = ({ _window } = {}) => (state) => ({
```

Comment:
I renamed it to `setupMapStateToProps`. I want to keep the name `mapStateToProps` since that's what we call it elsewhere.